src/evp/predictors.py

```diff
@@ -55,7 +56,7 @@ def predict(self, X) -> list[MerchantSizeByDPV]:
     def train(
         self, X_train, y_train, X_test, y_test, epochs=1, batch_size=None
     ) -> None:
-        log.info(f"Training {type(self).__name__}")
+        log.info(f"Training {type(self).__name__} for {epochs} epochs")
```

Comment:
Shouldn't the conventional epochs for simpler ML to be always 1? I am not familiar with AdaBoost architecture

Reply:
Yes, you're right